<launch>

  <arg name="record_data" default="true" />

  <!-- "1" = robot go straight in a row, "2" = round trip, "3" = robot navigate through all the rows -->
  <arg name="task" default="3" />

  <!-- For tasks 1 and 2 only, refers to the index of the row you want to use -->
  <arg name="rowNumber" default="1" />


  <node pkg="rasberry_navigation" name="navigation_testcase" type="navigation_testcase.py" output="screen" >
    <param name="task" value="$(arg task)" />
    <param name="rowNumber" value="$(arg rowNumber)" />
  </node>

  <node pkg="rosbag" type="record" name="rosbag_record_topoNav"
       args="record -o $(find rasberry_navigation)/topoNav_testcase/bags/ /odom /scan /tf /robot_pose -e /topological_(.*)/(.*)"
       if="$(arg record_data)" />

</launch>
